PyMca5/tests/ut/test_tiled_catalog_selector.py

```diff
@@ -72,7 +78,7 @@ def test_on_url_editing_finished():
     assert model.url == expected_url
     assert model._url_buffer == expected_url
 
-    mock_signal.emit.assert_called_once_with()
+    getattr(mock_signal.emit, expected_emit)()
```

Comment:
What do you think about making this a self-documenting assertion?

```suggestion
    assert_expected_emit = getattr(mock_signal.emit, expected_emit)
    assert_expected_emit()
```